Source/Foundation/bsfUtility/Error/BsCrashHandler.h

```diff
@@ -2,11 +2,37 @@
 //*********** Licensed under the MIT license. See LICENSE.md for full terms. This notice is not to be removed. ***********//
 #pragma once
 
+#include <functional>
+
 #define BS_MAX_STACKTRACE_DEPTH 200
 #define BS_MAX_STACKTRACE_NAME_BYTES 1024
 
 namespace bs
 {
+	/** Struct for holding crash handler settings */
+	struct CrashHandlerSettings
+	{
+		/** Called at the start of a when a windows SEH exception is started to be handled. Return true to skip default action */
```

Comment:
Comment seems to imply this is windows only

Reply:
Whoops, I forgot to write the first part of the sentence, it should say both for normal exception handling and SEH.